src/main/java/gov/usgs/wma/statistics/app/SwaggerConfig.java

```diff
@@ -0,0 +1,81 @@
+	public static final String StatsService_EXAMPLE_ENCODED="data=2005-06-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2006-06-10T04%3A15%3A00-05%3A00%2C+22.000%0D%0A2007-06-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2008-06-10T04%3A15%3A00-05%3A00%2C+2.000%0D%0A2009-06-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2010-06-10T04%3A15%3A00-05%3A00%2C+20.000%0D%0A2011-06-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2012-06-10T04%3A15%3A00-05%3A00%2C+2.000%0D%0A2013-06-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2014-06-10T04%3A15%3A00-05%3A00%2C+10.000%0D%0A2015-06-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2016-06-10T04%3A15%3A00-05%3A00%2C+43.000%0D%0A2017-06-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2018-06-10T04%3A15%3A00-05%3A00%2C+11.000%0D%0A2005-07-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2006-07-10T04%3A15%3A00-05%3A00%2C+22.000%0D%0A2007-07-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2008-07-10T04%3A15%3A00-05%3A00%2C+2.000%0D%0A2009-07-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2010-07-10T04%3A15%3A00-05%3A00%2C+20.000%0D%0A2011-07-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2012-07-10T04%3A15%3A00-05%3A00%2C+2.000%0D%0A2013-07-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2014-07-10T04%3A15%3A00-05%3A00%2C+10.000%0D%0A2015-07-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2016-07-10T04%3A15%3A00-05%3A00%2C+43.000%0D%0A2017-07-10T04%3A15%3A00-05%3A00%2C+1.000%0D%0A2018-06-10T04%3A15%3A00-05%3A00%2C+11.000%0D%0A%09%09%09";
+	public static final String StatsService_EXAMPLE_RAW
+			="2005-06-10T04:15:00-05:00, 1.000\n"
+			+"2006-06-10T04:15:00-05:00, 22.000\n"
+			+"2007-06-10T04:15:00-05:00, 1.000\n"
+			+"2008-06-10T04:15:00-05:00, 2.000\n"
+			+"2009-06-10T04:15:00-05:00, 1.000\n"
+			+"2010-06-10T04:15:00-05:00, 20.000\n"
+			+"2011-06-10T04:15:00-05:00, 1.000\n"
+			+"2012-06-10T04:15:00-05:00, 2.000\n"
+			+"2013-06-10T04:15:00-05:00, 1.000\n"
+			+"2014-06-10T04:15:00-05:00, 10.000\n"
+			+"2015-06-10T04:15:00-05:00, 1.000\n"
+			+"2016-06-10T04:15:00-05:00, 43.000\n"
+			+"2017-06-10T04:15:00-05:00, 1.000\n"
+			+"2018-06-10T04:15:00-05:00, 11.000\n"
+			+"2005-07-10T04:15:00-05:00, 1.000\n"
+			+"2006-07-10T04:15:00-05:00, 22.000\n"
+			+"2007-07-10T04:15:00-05:00, 1.000\n"
+			+"2008-07-10T04:15:00-05:00, 2.000\n"
+			+"2009-07-10T04:15:00-05:00, 1.000\n"
+			+"2010-07-10T04:15:00-05:00, 20.000\n"
+			+"2011-07-10T04:15:00-05:00, 1.000\n"
+			+"2012-07-10T04:15:00-05:00, 2.000\n"
+			+"2013-07-10T04:15:00-05:00, 1.000\n"
+			+"2014-07-10T04:15:00-05:00, 10.000\n"
+			+"2015-07-10T04:15:00-05:00, 1.000\n"
+			+"2016-07-10T04:15:00-05:00, 43.000\n"
+			+"2017-07-10T04:15:00-05:00, 1.000\n"
+			+"2018-06-10T04:15:00-05:00, 11.000\n";
+	
+	
+	@Autowired
+	private TypeResolver typeResolver;
+	
+	@Bean
+	public Docket qwPortalServicesApi() {
+		Docket docket = new Docket(DocumentationType.SWAGGER_2)
+//			.protocols(new HashSet<>(Arrays.asList("https")))
+//			.useDefaultResponseMessages(false)
+//			.host(configurationService.getSwaggerDisplayHost())
+//			.pathProvider(pathProvider())
```

Comment:
Delete comments.

Reply:
Yep, this was the first commit (ie not in the history) and I wanted to keep for a time the "documentation" of springfox for time. That said, it should go it not used.